### PR TITLE
Mongodb crud operations examples added

### PR DESCRIPTION
**Pull request by kashifkhan0771**

### CODE REVIEW

File: mongoDB/main.go [hunk 1]
```diff
@@ -0,0 +1,235 @@
+package main
+
+import (
+	"bufio"
+	"context"
+	"fmt"
+	"go.mongodb.org/mongo-driver/bson"
```
Comment (shahzadhaider1) [inline mongoDB/main.go]: imports not sorted

**Review by shahzadhaider1 — CHANGES_REQUESTED** [review]
minor change needed. BTW, did you run it and checked with mongodb?

File: mongoDB/main.go (hunk 1 above)
Comment (kashifkhan0771) [reply mongoDB/main.go]: Yes all operations are executed successfully with local mongodb

Comment (shahzadhaider1) [reply mongoDB/main.go]: ok. good work

**Pull request merged**